src/modules/holder/pages/Home/Home.tsx

```diff
@@ -41,21 +40,49 @@ export const Home: FC = () => {
       </>
     )
   }
+
+  const validTickets = data.filter((credentialItem) => {
+    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
+    if (Date.parse(credentialSubject?.eventDate) > Date.now()) {
+      return credentialItem
+    }
+  })
```

Comment:
`filter` expecting boolean value
```suggestion
const validTickets = data.filter((credentialItem) => {
    const credentialSubject = (credentialItem as StoredW3CCredential)?.credentialSubject
    return Date.parse(credentialSubject?.eventDate) > Date.now()
  })
```